sark/code/segment.py

```diff
@@ -244,6 +247,11 @@ def __repr__(self):
                                         self.bitness)
 
 
-def segments():
+def segments(seg_type=None):
+    ''' :param seg_type: type of segment e.g. SEG_CODE
+        :return: segment based on type. if type is None, returns all segments
```

Comment:
I try to follow this - https://pypi.python.org/pypi/sphinxcontrib-napoleon - in the docstrings, as it is more readable for devs, and gets parse just as well.